source/GameLoadingPanel.cpp

```diff
@@ -65,6 +64,13 @@ void GameLoadingPanel::Step()
 		GetUI()->Push(new MenuPanel(player, gamePanels));
```

Comment:
I wonder, do we need to show the menu panel if there is a conversation?

Reply:
Probably not; it looks a little cleaner, too.

![Screenshot taken at 2022-04-16_13-05-00](https://user-images.githubusercontent.com/875669/163684526-9c5dce2b-a821-47ad-a87c-7f4673ddb142.png)

Reply:
Fixed